Common/src/main/resources/lang/lang_fr.yml

```diff
@@ -32,13 +32,13 @@ import:
   end: "<c4>Les données ont été importées avec succès!</c4>"
 
 kick:
-  api_mode: "(Plz translate) The plugin is in API mode, and no actions have been performed."
-  end_vpn: "(Plz translate) <c4>Player</c4> {player} <c4>has been kicked for VPN usage, and/or all necessary commands have been run.</c4>"
-  end_mcleaks: "(Plz translate) <c4>Player</c4> {player} <c4>has been kicked for MCLeaks usage, and/or and all necessary commands have been run.</c4>"
-  no_player: "(Plz translate) The specified player was not online."
+  api_mode: "Le plugin est en mode API, et aucune action a été effectuée."
+  end_vpn: "<c4>Le joueur</c4> {player} <c4>a été éjecté pour une utilisation d'un VPN, et/ou toutes les commandes ont été exécutées.</c4>"
+  end_mcleaks: " <c4>Le joueur</c4> {player} <c4>a été éjecté pour une utilisation MCLeaks, et/ou toutes les commandes nécessaires ont été exécutées.</c4>"
```

Comment:
Random space at the beginning of the line

Reply:
Fixed